commands/mr/create/mr_create.go

```diff
@@ -155,6 +156,7 @@ func NewCmdCreate(f *cmdutils.Factory, runE func(opts *CreateOpts) error) *cobra
 	mrCreateCmd.Flags().StringVarP(&opts.MilestoneFlag, "milestone", "m", "", "The global ID or title of a milestone to assign")
 	mrCreateCmd.Flags().BoolVarP(&opts.AllowCollaboration, "allow-collaboration", "", false, "Allow commits from other members")
 	mrCreateCmd.Flags().BoolVarP(&opts.RemoveSourceBranch, "remove-source-branch", "", false, "Remove Source Branch on merge")
+	mrCreateCmd.Flags().BoolVarP(&opts.Squash, "squash", "", false, "Squash commits into a single commit when merging")
```

Comment:
I'm not quite certain whether to go with `--squash` or `--squash-before-merge`

Reply:
Agree, I'll rename it :)